tests/handlers/test_combined_data.py

```diff
@@ -7,58 +7,104 @@
 
 
 def test_load(va_governor_county_data):
+    election_id = "2017-11-07_VA_G"
+    office_id = "G"
+    geographic_unit_type = "county"
     estimands = ["turnout"]
+    estimand_baselines = {"turnout": "turnout"}
     live_data_handler = MockLiveDataHandler(
-        "2017-11-07_VA_G", "G", "county", estimands=["turnout"], data=va_governor_county_data
+        election_id, office_id, geographic_unit_type, estimands=["turnout"], data=va_governor_county_data
     )
     current_data = live_data_handler.data
+    preprocessed_data_handler = PreprocessedDataHandler(
+        election_id,
+        office_id,
+        geographic_unit_type,
+        estimand_baselines=estimand_baselines,
+        estimands=estimands,
+        data=va_governor_county_data,
+    )
+    preprocessed_data = preprocessed_data_handler.data
 
     combined_data_handler = CombinedDataHandler(
-        va_governor_county_data, current_data, estimands, "county", handle_unreporting="drop"
+        preprocessed_data, current_data, estimands, "county", handle_unreporting="drop"
     )
-    assert combined_data_handler.data.shape == (133, 29)
+
+    assert combined_data_handler.data.shape == (133, 33)
```

Comment:
I'm seeing increases of 4-6 columns for a lot of unit tests like this one.  Can you explain why?  I'm trying to think through the columns that are being added now.  Possibly `baseline_turnout` and `results_turnout` but that's all I can think of 🤔

Reply:
yes! it's turnout factor and results turnout and the two weights columns (results and baseline)